.tekton/pull-request.yaml

```diff
@@ -37,12 +37,25 @@ spec:
         taskRef:
           bundle: quay.io/redhat-appstudio/appstudio-tasks:510fa6e99f1fa1f816c96354bbaf1ad155c6d9c3-1
           name: git-clone
-      - name: run-tests
+      - name: lint-prometheus-rules
         workspaces:
           - name: workspace
             workspace: workspace
         runAfter:
           - clone-repository
+        taskSpec:
+          workspaces:
+            - name: workspace
+          steps:
+            - name: lint-rules
+              image: registry.access.redhat.com/ubi8/ubi:latest
+              script: cd $(workspaces.workspace.path) && ./automation/lint_prometheus-rules.sh
```

Comment:
This works but you can also change the starting directory using the `workingDir` field for a step.

Reply:
Indeed. I've changed it to work with `workingDir`